dockerfiles/Synapse.Dockerfile

```diff
@@ -24,7 +24,7 @@ COPY keys/* /ca/
 RUN openssl genrsa -out /conf/server.tls.key 2048
 
 # generate a signing key
-RUN generate_signing_key -o /conf/server.signing.key
+RUN /synapse/.venv/bin/generate_signing_key -o /conf/server.signing.key
```

Comment:
can we arrange for `/synapse/.venv/bin` to be on the default PATH, so we don't need to worry about this?